docs/tutorial/metrics.rst

```diff
@@ -0,0 +1,76 @@
+Collect Metrics from MicroCeph using Prometheus
+================================================
+
+Introduction
+~~~~~~~~~~~~
```

Comment:
Kindly follow the style guide for (underlining/formatting) RST headers.

https://docs.openstack.org/charm-guide/latest/community/doc-contrib/rst.html#section-headers